index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Fraunces:ital,opsz,wght@0,9..144,100..900;1,9..144,100..900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer ></script>
+</head>
+<body>
+    <main>
+        <header>
+            <nav class="nav-bar">
+                    <p class="name">Adebisi.A</p>
+                    <div class="cards">
+                        <button type="button">About</button>
+                        <button type="button">Projects</button>
+                        <button type="button">Skills</button>
+                        <button type="button">Contact</button>
+                    </div>
+                </nav>
+
+        </header>
+        <section class="introduction" >
+            
+                <div>
+                    <img class="photo" src="assets/myself.jpg" alt="my-picture">
+                </div>
+               
+                <div class="intro">
+                        <p class="intro1">I'm Adeboye.Y.Adebisi <br>
+                            Front-end Developer <br>
+                            based in Nigeria. 
+                        </p>
+                        <p class="intro2" >A passionate designer,delivering the 
+                            best user-friendly Web pages,An entrepreneur who 
+                            loves customers' satisfaction and making profits.
```

Comment:
<img width="1472" alt="image" src="https://github.com/user-attachments/assets/d1b7362f-42ab-4c62-ae6e-5dbb1bd4195f" />

Kindly see that this text is placed/wrapped within the container. Ensure your screen resolution/zoom is at 100% when you do this.